sessionctx/variable/varsutil.go

```diff
@@ -348,6 +348,8 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
 		return value, ErrWrongValueForVar.GenWithStackByArgs(name, value)
 	case SQLSelectLimit:
 		return checkUInt64SystemVar(name, value, 0, math.MaxUint64, vars)
+	case TiDBStoreLimit:
+		return checkInt64SystemVar(name, value, 0, math.MaxInt64, vars)
```

Comment:
Do we need to add an upper limit？

Reply:
hard to define a valid upper limit (big table scanning can cause store limit increase). now is math.MaxInt64.